### PR TITLE
feat: (Delete Asignaturas) metodo de borrar asignatura

### PR DESCRIPTION
**Pull request by PaulAnds1**

### CODE REVIEW

File: packages/solicitudes/src/use-cases/db/asignaturas/delete.asignatura.use-cases.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+) => {
+  const asignatura2 = await findOneAsignaturaQuery(identifierObj);
+  checkers.throwErrorIfDataIsFalsy(asignatura2, 'asignaturas', identifierObj.id);
+
+  Logger.info('[asignatura/delete]: Deleting asignatura');
+  const asignatura = await deleteAsignaturaQuery(identifierObj);
+  Logger.info('[asignatura/delete]: Asignatura deleted');
+
+  return asignatura;
+};
```
Comment (robertotorres-dev) [inline packages/solicitudes/src/use-cases/db/asignaturas/delete.asignatura.use-cases.js:15]: you can change the first constant by asignatura
and the second constant by asignaturaDeleted

File: packages/solicitudes/src/use-cases/index.js [hunk 2]
```diff
@@ -3,7 +3,6 @@ const {
   findAllSolicitudesProgramas,
   findOneSolicitudPrograma,
   findSolicitudesUsuario,
-  updateSolicitudPrograma,
 } = require('./db/solicitudes');
```
Comment (robertotorres-dev) [inline packages/solicitudes/src/use-cases/index.js:7]: you don't need to delete use cases already created

File: packages/solicitudes/src/use-cases/index.js [hunk 3]
```diff
@@ -27,6 +30,6 @@ module.exports = {
   updateDiligencia,
   deleteDiligencia,
   createAsignaturaPrograma,
-  updateSolicitudPrograma,
   findOneAsignatura,
```
Comment (robertotorres-dev) [inline packages/solicitudes/src/use-cases/index.js:33]: you don't need to delete use cases already created

File: packages/api-gateway/src/drivers/http/routes/asignaturas/.DS_Store
Comment (robertotorres-dev) [inline packages/api-gateway/src/drivers/http/routes/asignaturas/.DS_Store:1]: What is this?

Comment (PaulAnds1) [reply packages/api-gateway/src/drivers/http/routes/asignaturas/.DS_Store:1]: Its a file thats created every time i open an archive in mac, i may have missed deleting that one, ill get to it

**Pull request merged**